components/post_view/post_flag_icon/post_flag_icon.js

```diff
@@ -83,7 +83,7 @@ export default class PostFlagIcon extends React.PureComponent {
                 }
             >
                 <button
-                    id={flagIconId}
+                    id={flagIconId || `${this.props.idPrefix}_${this.props.postId}`}
```

Comment:
This is to keep backward compatibility with existing Se tests.  This is be addressed separately by [MM-14242](https://mattermost.atlassian.net/browse/MM-14242).